protocol/communities/manager.go

```diff
@@ -2838,7 +2838,7 @@ func (m *Manager) HandleCommunityRequestToJoinResponse(signer *ecdsa.PublicKey,
 		return nil, err
 	}
 
-	isControlNodeSigner := common.IsPubKeyEqual(community.PublicKey(), signer)
+	isControlNodeSigner := common.IsPubKeyEqual(community.ControlNode(), signer)
```

Comment:
What does `community.ControlNode()` return when the Owner token is not minted yet?

Reply:
config.ID which is PublicKey of the community